Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -216,6 +314,82 @@ class FavoritesViewController: UIViewController, Themeable {
         collection.collectionViewLayout.invalidateLayout()
     }
     
+    private func setupBackgroundImage() {
+        guard var background = backgroundImage.info,
+            let image = background.image else {
+            return
+        }
+        
+        let imageAspectRatio = image.size.width / image.size.height
+        let imageView = UIImageView(image: image)
+        
+        imageView.contentMode = UIImageView.ContentMode.scaleAspectFit
+        // Make sure it goes to the back
+        view.insertSubview(imageView, at: 0)
+        
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.snp.makeConstraints {
+            
+            // Determines the height of the content
+            // `999` priority is required for landscape, since top/bottom constraints no longer the most important
+            //   using `high` is not enough either.
+            $0.top.bottom.equalToSuperview().priority(ConstraintPriority(999))
```

Comment:
Would using `.required` here instead of `999` work? or is it too strong?
Required has a value of `1000`.

Reply:
Too strong, since in landscape, these constraints are overridden by `centerY`, which takes precedence and nicely crops the top and bottom of the image. Will leave better comment.